deploy-precise-code-intel-worker.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+source ./replicas.sh
+
+# Description: Handles conversion of uploaded precise code intelligence bundles.
+#
+# Ports exposed to other Sourcegraph services: 3188/TCP (first worker), 9090 (prometheus)
+# Ports exposed to the public internet: none
+#
+docker run --detach \
+    --name=precise-code-intel-worker \
+    --network=sourcegraph \
+    --restart=always \
+    --cpus=2 \
+    --memory=4g \
+    -e 'NUM_WORKERS=2' \
```

Comment:
shouldn't be here, we should be able to rely on the default being 2

Reply:
Do you mean the default in the docker container?

Reply:
Yes

Reply:
I'm removing all NUM_WORKERS from this repo and will change the image default to 2 in sg/sg.